### PR TITLE
Bump sentry to 5.7.4

### PR DESCRIPTION
**Pull request by m-kuhn**

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for e27b3647ade7b04e741b620f451302ba13adc6ef: [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3578-e90b0497779da5921a294132cab515b34125f611-arm64-android.apk)
<details>
<summary>Other architectures</summary>

- [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3578-e90b0497779da5921a294132cab515b34125f611-arm-android.apk)
- [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3578-e90b0497779da5921a294132cab515b34125f611-x64-android.apk)
- [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3578-e90b0497779da5921a294132cab515b34125f611-x86-android.apk)
- [**all access arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-3578-e90b0497779da5921a294132cab515b34125f611-arm64-android.apk)
- [all access arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-3578-e90b0497779da5921a294132cab515b34125f611-arm-android.apk)
- [all access x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-3578-e90b0497779da5921a294132cab515b34125f611-x64-android.apk)
- [all access x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-3578-e90b0497779da5921a294132cab515b34125f611-x86-android.apk)
</details>

**Comment by m-kuhn** [comment]
Crashes for me after showing the welcome screen

**Comment by nirvn** [comment]
Same here; logcat's crash backtrace not super helpful:

```
--------- beginning of crash
11-07 09:12:19.490 15117 15146 F libc    : Fatal signal 11 (SIGSEGV), code 2 (SEGV_ACCERR), fault addr 0x757fe898b8 in tid 15146 (qtMainLoopThrea), pid 15117 (gis.qfield_beta)
11-07 09:12:19.651 15193 15193 I crash_dump64: obtaining output fd from tombstoned, type: kDebuggerdTombstoneProto
11-07 09:12:19.656   672   672 I tombstoned: received crash request for pid 15146
11-07 09:12:19.657 15193 15193 I crash_dump64: performing dump of process 15117 (target tid = 15146)
11-07 09:12:20.371 15193 15193 F DEBUG   : *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
11-07 09:12:20.371 15193 15193 F DEBUG   : Build fingerprint: 'google/bluejay/bluejay:13/TP1A.221005.003/9018821:user/release-keys'
11-07 09:12:20.371 15193 15193 F DEBUG   : Revision: 'MP1.0'
11-07 09:12:20.371 15193 15193 F DEBUG   : ABI: 'arm64'
11-07 09:12:20.372 15193 15193 F DEBUG   : Timestamp: 2022-11-07 09:12:19.685586298+0700
11-07 09:12:20.372 15193 15193 F DEBUG   : Process uptime: 4s
11-07 09:12:20.372 15193 15193 F DEBUG   : Cmdline: ch.opengis.qfield_beta
11-07 09:12:20.372 15193 15193 F DEBUG   : pid: 15117, tid: 15146, name: qtMainLoopThrea  >>> ch.opengis.qfield_beta <<<
11-07 09:12:20.372 15193 15193 F DEBUG   : uid: 10023
11-07 09:12:20.372 15193 15193 F DEBUG   : tagged_addr_ctrl: 0000000000000001 (PR_TAGGED_ADDR_ENABLE)
11-07 09:12:20.372 15193 15193 F DEBUG   : signal 11 (SIGSEGV), code 2 (SEGV_ACCERR), fault addr 0x000000757fe898b8
11-07 09:12:20.372 15193 15193 F DEBUG   :     x0  000000744c987ad0  x1  000000744c986de8  x2  0000000000000001  x3  b40000754ffdabf0
11-07 09:12:20.372 15193 15193 F DEBUG   :     x4  000000744c987ad0  x5  0000007427d5f964  x6  feff0270a2649b0a  x7  7f7f7f7fff7fff7f
11-07 09:12:20.372 15193 15193 F DEBUG   :     x8  b40000757fe898b8  x9  0000000000000003  x10 0000000000003b2a  x11 0000000000003b2a
11-07 09:12:20.372 15193 15193 F DEBUG   :     x12 0000000000000001  x13 0000000000000001  x14 ffffffffffffffff  x15 0000000000000001
11-07 09:12:20.372 15193 15193 F DEBUG   :     x16 0000007445fd18a0  x17 0000007445d53f68  x18 000000744c686000  x19 000000744c986de8
11-07 09:12:20.372 15193 15193 F DEBUG   :     x20 000000744c987ad0  x21 000000744c98b000  x22 0000000000000001  x23 000000744c987ad0
11-07 09:12:20.372 15193 15193 F DEBUG   :     x24 0000000000000000  x25 000000742f184540  x26 b40000757fe898b8  x27 000000742aa980cc
11-07 09:12:20.372 15193 15193 F DEBUG   :     x28 000000742a920175  x29 000000744c986dc0
11-07 09:12:20.372 15193 15193 F DEBUG   :     lr  0000007427d6342c  sp  000000744c986dc0  pc  000000757fe898b8  pst 0000000000001000
11-07 09:12:20.372 15193 15193 F DEBUG   : backtrace:
11-07 09:12:20.372 15193 15193 F DEBUG   :       #00 pc 000000000013e8b8  [anon:scudo:primary]
11-07 09:12:20.372 15193 15193 F DEBUG   :       #01 pc 0000000000162428  /data/app/~~tv_UfoQ7cHQsi84AuVRWlA==/ch.opengis.qfield_beta-s1gJxAql8PSPSCPY5280Ag==/lib/arm64/libsentry.so (BuildId: 5c2f2cb0ff7717720760a183997c8f6e8e613378)
11-07 09:12:20.372 15193 15193 F DEBUG   :       #02 pc 000000000016205c  /data/app/~~tv_UfoQ7cHQsi84AuVRWlA==/ch.opengis.qfield_beta-s1gJxAql8PSPSCPY5280Ag==/lib/arm64/libsentry.so (BuildId: 5c2f2cb0ff7717720760a183997c8f6e8e613378)
11-07 09:12:20.372 15193 15193 F DEBUG   :       #03 pc 000000000015ebf0  /data/app/~~tv_UfoQ7cHQsi84AuVRWlA==/ch.opengis.qfield_beta-s1gJxAql8PSPSCPY5280Ag==/lib/arm64/libsentry.so (BuildId: 5c2f2cb0ff7717720760a183997c8f6e8e613378)
11-07 09:12:20.372 15193 15193 F DEBUG   :       #04 pc 000000000015ea44  /data/app/~~tv_UfoQ7cHQsi84AuVRWlA==/ch.opengis.qfield_beta-s1gJxAql8PSPSCPY5280Ag==/lib/arm64/libsentry.so (__gxx_personality_v0+224) (BuildId: 5c2f2cb0ff7717720760a183997c8f6e8e613378)
11-07 09:12:20.372 15193 15193 F DEBUG   :       #05 pc 00000000002eb32c  /data/app/~~tv_UfoQ7cHQsi84AuVRWlA==/ch.opengis.qfield_beta-s1gJxAql8PSPSCPY5280Ag==/lib/arm64/libQt5Core_arm64-v8a.so (BuildId: 277ca13e37d037dac7836b411ab37cc43cc4cba8)
11-07 09:12:20.372 15193 15193 F DEBUG   :       #06 pc 00000000002eb834  /data/app/~~tv_UfoQ7cHQsi84AuVRWlA==/ch.opengis.qfield_beta-s1gJxAql8PSPSCPY5280Ag==/lib/arm64/libQt5Core_arm64-v8a.so (_Unwind_Resume+116) (BuildId: 277ca13e37d037dac7836b411ab37cc43cc4cba8)
11-07 09:12:20.372 15193 15193 F DEBUG   :       #07 pc 000000000461a740  /data/app/~~tv_UfoQ7cHQsi84AuVRWlA==/ch.opengis.qfield_beta-s1gJxAql8PSPSCPY5280Ag==/lib/arm64/libqfield_arm64-v8a.so (BuildId: 7d2893cbf146652ddb78c54139a3bbf16be006ef)
11-07 09:12:20.372 15193 15193 F DEBUG   :       #08 pc 0000000004605344  /data/app/~~tv_UfoQ7cHQsi84AuVRWlA==/ch.opengis.qfield_beta-s1gJxAql8PSPSCPY5280Ag==/lib/arm64/libqfield_arm64-v8a.so (BuildId: 7d2893cbf146652ddb78c54139a3bbf16be006ef)
11-07 09:12:20.372 15193 15193 F DEBUG   :       #09 pc 00000000045cf7bc  /data/app/~~tv_UfoQ7cHQsi84AuVRWlA==/ch.opengis.qfield_beta-s1gJxAql8PSPSCPY5280Ag==/lib/arm64/libqfield_arm64-v8a.so (BuildId: 7d2893cbf146652ddb78c54139a3bbf16be006ef)
11-07 09:12:20.372 15193 15193 F DEBUG   :       #10 pc 00000000045edbe4  /data/app/~~tv_UfoQ7cHQsi84AuVRWlA==/ch.opengis.qfield_beta-s1gJxAql8PSPSCPY5280Ag==/lib/arm64/libqfield_arm64-v8a.so (BuildId: 7d2893cbf146652ddb78c54139a3bbf16be006ef)
11-07 09:12:20.372 15193 15193 F DEBUG   :       #11 pc 00000000045f34a8  /data/app/~~tv_UfoQ7cHQsi84AuVRWlA==/ch.opengis.qfield_beta-s1gJxAql8PSPSCPY5280Ag==/lib/arm64/libqfield_arm64-v8a.so (BuildId: 7d2893cbf146652ddb78c54139a3bbf16be006ef)
11-07 09:12:20.372 15193 15193 F DEBUG   :       #12 pc 0000000004678d70  /data/app/~~tv_UfoQ7cHQsi84AuVRWlA==/ch.opengis.qfield_beta-s1gJxAql8PSPSCPY5280Ag==/lib/arm64/libqfield_arm64-v8a.so (BuildId: 7d2893cbf146652ddb78c54139a3bbf16be006ef)
11-07 09:12:20.372 15193 15193 F DEBUG   :       #13 pc 0000000002b77d0c  /data/app/~~tv_UfoQ7cHQsi84AuVRWlA==/ch.opengis.qfield_beta-s1gJxAql8PSPSCPY5280Ag==/lib/arm64/libqfield_arm64-v8a.so (BuildId: 7d2893cbf146652ddb78c54139a3bbf16be006ef)
11-07 09:12:20.372 15193 15193 F DEBUG   :       #14 pc 0000000002b72d88  /data/app/~~tv_UfoQ7cHQsi84AuVRWlA==/ch.opengis.qfield_beta-s1gJxAql8PSPSCPY5280Ag==/lib/arm64/libqfield_arm64-v8a.so (BuildId: 7d2893cbf146652ddb78c54139a3bbf16be006ef)
11-07 09:12:20.372 15193 15193 F DEBUG   :       #15 pc 0000000002bb3bc4  /data/app/~~tv_UfoQ7cHQsi84AuVRWlA==/ch.opengis.qfield_beta-s1gJxAql8PSPSCPY5280Ag==/lib/arm64/libqfield_arm64-v8a.so (BuildId: 7d2893cbf146652ddb78c54139a3bbf16be006ef)
11-07 09:12:20.372 15193 15193 F DEBUG   :       #16 pc 0000000002bb53d0  /data/app/~~tv_UfoQ7cHQsi84AuVRWlA==/ch.opengis.qfield_beta-s1gJxAql8PSPSCPY5280Ag==/lib/arm64/libqfield_arm64-v8a.so (BuildId: 7d2893cbf146652ddb78c54139a3bbf16be006ef)
11-07 09:12:20.372 15193 15193 F DEBUG   :       #17 pc 000000000013e67c  /data/app/~~tv_UfoQ7cHQsi84AuVRWlA==/ch.opengis.qfield_beta-s1gJxAql8PSPSCPY5280Ag==/lib/arm64/libsentry.so (std::__ndk1::__call_once(unsigned long volatile&, void*, void (*)(void*))+164) (BuildId: 5c2f2cb0ff7717720760a183997c8f6e8e613378)
11-07 09:12:20.372 15193 15193 F DEBUG   :       #18 pc 0000000002bae844  /data/app/~~tv_UfoQ7cHQsi84AuVRWlA==/ch.opengis.qfield_beta-s1gJxAql8PSPSCPY5280Ag==/lib/arm64/libqfield_arm64-v8a.so (BuildId: 7d2893cbf146652ddb78c54139a3bbf16be006ef)
11-07 09:12:20.372 15193 15193 F DEBUG   :       #19 pc 00000000026960c4  /data/app/~~tv_UfoQ7cHQsi84AuVRWlA==/ch.opengis.qfield_beta-s1gJxAql8PSPSCPY5280Ag==/lib/arm64/libqfield_arm64-v8a.so (BuildId: 7d2893cbf146652ddb78c54139a3bbf16be006ef)
11-07 09:12:20.372 15193 15193 F DEBUG   :       #20 pc 000000000281f900  /data/app/~~tv_UfoQ7cHQsi84AuVRWlA==/ch.opengis.qfield_beta-s1gJxAql8PSPSCPY5280Ag==/lib/arm64/libqfield_arm64-v8a.so (BuildId: 7d2893cbf146652ddb78c54139a3bbf16be006ef)
11-07 09:12:20.372 15193 15193 F DEBUG   :       #21 pc 0000000002116c90  /data/app/~~tv_UfoQ7cHQsi84AuVRWlA==/ch.opengis.qfield_beta-s1gJxAql8PSPSCPY5280Ag==/lib/arm64/libqfield_arm64-v8a.so (BuildId: 7d2893cbf146652ddb78c54139a3bbf16be006ef)
11-07 09:12:20.372 15193 15193 F DEBUG   :       #22 pc 0000000002109d28  /data/app/~~tv_UfoQ7cHQsi84AuVRWlA==/ch.opengis.qfield_beta-s1gJxAql8PSPSCPY5280Ag==/lib/arm64/libqfield_arm64-v8a.so (BuildId: 7d2893cbf146652ddb78c54139a3bbf16be006ef)
11-07 09:12:20.372 15193 15193 F DEBUG   :       #23 pc 000000000020f3cc  /data/app/~~tv_UfoQ7cHQsi84AuVRWlA==/ch.opengis.qfield_beta-s1gJxAql8PSPSCPY5280Ag==/lib/arm64/libQt5Core_arm64-v8a.so (BuildId: 277ca13e37d037dac7836b411ab37cc43cc4cba8)
11-07 09:12:20.372 15193 15193 F DEBUG   :       #24 pc 00000000002133a0  /data/app/~~tv_UfoQ7cHQsi84AuVRWlA==/ch.opengis.qfield_beta-s1gJxAql8PSPSCPY5280Ag==/lib/arm64/libQt5Core_arm64-v8a.so (QTimer::timeout(QTimer::QPrivateSignal)+56) (BuildId: 277ca13e37d037dac7836b411ab37cc43cc4cba8)
11-07 09:12:20.372 15193 15193 F DEBUG   :       #25 pc 0000000000208560  /data/app/~~tv_UfoQ7cHQsi84AuVRWlA==/ch.opengis.qfield_beta-s1gJxAql8PSPSCPY5280Ag==/lib/arm64/libQt5Core_arm64-v8a.so (QObject::event(QEvent*)+120) (BuildId: 277ca13e37d037dac7836b411ab37cc43cc4cba8)
11-07 09:12:20.372 15193 15193 F DEBUG   :       #26 pc 000000000018f640  /data/app/~~tv_UfoQ7cHQsi84AuVRWlA==/ch.opengis.qfield_beta-s1gJxAql8PSPSCPY5280Ag==/lib/arm64/libQt5Widgets_arm64-v8a.so (QApplicationPrivate::notify_helper(QObject*, QEvent*)+304) (BuildId: 7dd8ce81350d9e9e7c4970507de9f3995599a182)
```

File: .github/workflows/android.yml [hunk 1]
```diff
@@ -67,7 +67,7 @@ jobs:
       - name: Download sentry libs
         run: |
           mkdir /tmp/sentry-android-ndk
-          wget https://repo1.maven.org/maven2/io/sentry/sentry-android-ndk/5.5.2/sentry-android-ndk-5.5.2.aar -O /tmp/sentry.zip
+          wget https://repo1.maven.org/maven2/io/sentry/sentry-android-ndk/6.7.0/sentry-android-ndk-6.7.0.aar -O /tmp/sentry.zip
```
Comment (nirvn) [inline .github/workflows/android.yml:70]:
```suggestion
          wget https://repo1.maven.org/maven2/io/sentry/sentry-android-ndk/5.7.4/sentry-android-ndk-5.7.4.aar -O /tmp/sentry.zip
```

File: platform/android/build.gradle.in [hunk 2]
```diff
@@ -119,5 +119,5 @@ android {
 
 // Add Sentry's SDK as a dependency.
 dependencies {
-    implementation 'io.sentry:sentry-android:5.5.2'
+    implementation 'io.sentry:sentry-android:6.7.0'
```
Comment (nirvn) [inline platform/android/build.gradle.in:122]:
```suggestion
    implementation 'io.sentry:sentry-android:5.7.4'
```

**Comment by nirvn** [comment]
@m-kuhn , 5.7.4 doesn't crash for me.

File: .github/workflows/android.yml [hunk 3]
```diff
@@ -67,7 +67,7 @@ jobs:
       - name: Download sentry libs
         run: |
           mkdir /tmp/sentry-android-ndk
-          wget https://repo1.maven.org/maven2/io/sentry/sentry-android-ndk/5.5.2/sentry-android-ndk-5.5.2.aar -O /tmp/sentry.zip
+          wget https://repo1.maven.org/maven2/io/sentry/sentry-android-ndk/5.7.4/sentry-android-ndk-5.7.4.aar -O /tmp/sentry.zip
```
Comment (nirvn) [inline .github/workflows/android.yml:70]:
```suggestion
          wget https://repo1.maven.org/maven2/io/sentry/sentry-android-ndk/6.4.4/sentry-android-ndk-6.4.4.aar -O /tmp/sentry.zip
```

File: platform/android/build.gradle.in [hunk 4]
```diff
@@ -119,5 +119,5 @@ android {
 
 // Add Sentry's SDK as a dependency.
 dependencies {
-    implementation 'io.sentry:sentry-android:5.5.2'
+    implementation 'io.sentry:sentry-android:5.7.4'
```
Comment (nirvn) [inline platform/android/build.gradle.in:122]:
```suggestion
    implementation 'io.sentry:sentry-android:6.4.4'
```

**Comment by nirvn** [comment]
6.4.4 crash-free

**Comment by nirvn** [comment]
@m-kuhn , 6.4.4 is crash free, however I can't see manual log submissions (while I could with 5.7.4)

**Comment by m-kuhn** [comment]
> @m-kuhn , 6.4.4 is crash free, however I can't see manual log submissions (while I could with 5.7.4)

anything in the catlog?

**Comment by nirvn** [comment]
@m-kuhn , nothing. Can you try to install the latest APK (which is sentry 6.4.4) and try to submit a message log on your end?

File: .github/workflows/android.yml [hunk 5]
```diff
@@ -67,7 +67,7 @@ jobs:
       - name: Download sentry libs
         run: |
           mkdir /tmp/sentry-android-ndk
-          wget https://repo1.maven.org/maven2/io/sentry/sentry-android-ndk/5.5.2/sentry-android-ndk-5.5.2.aar -O /tmp/sentry.zip
+          wget https://repo1.maven.org/maven2/io/sentry/sentry-android-ndk/6.4.4/sentry-android-ndk-6.4.4.aar -O /tmp/sentry.zip
```
Comment (nirvn) [inline .github/workflows/android.yml:70]:
```suggestion
          wget https://repo1.maven.org/maven2/io/sentry/sentry-android-ndk/6.3.1/sentry-android-ndk-6.3.1.aar -O /tmp/sentry.zip
```

File: platform/android/build.gradle.in [hunk 6]
```diff
@@ -119,5 +119,5 @@ android {
 
 // Add Sentry's SDK as a dependency.
 dependencies {
-    implementation 'io.sentry:sentry-android:5.5.2'
+    implementation 'io.sentry:sentry-android:6.4.4'
```
Comment (nirvn) [inline platform/android/build.gradle.in:122]:
```suggestion
    implementation 'io.sentry:sentry-android:6.3.1'
```

**Review by nirvn — APPROVED** [review]

**Pull request merged**